src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -928,12 +904,9 @@ public function register($id, $class = null)
      * This method implements a shortcut for using setDefinition() with
      * an autowired definition.
      *
-     * @param string      $id    The service identifier
-     * @param string|null $class The service class
-     *
      * @return Definition The created definition
      */
-    public function autowire($id, $class = null)
+    public function autowire(string $id, string $class = null)
     {
         return $this->setDefinition($id, (new Definition($class))->setAutowired(true));
```

Comment:
`setAutowired` has been forgotten to add the type

Reply:
indeed